sniffles/ruletrafficgenerator.py

```diff
@@ -1718,7 +1718,9 @@ def get_dist_mac_oui(self):
                 break
             else:
                 last_key = i
-        if prefix is None:
+
+        # [] is not same as None
+        if len(prefix) == 0:
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) recommends to use
```
if not prefix:
```
instead of checking the length of a list.